### DISCUSSION TITLE
Should formulas include instructions/warnings for building --HEAD?

### DISCUSSION THREAD

**Discussion by okket** [Homebrew/discussions#740] [Everyday usage] [answered]
The current situation with `neovim` is that the current release 0.4.4 feels a bit ancient, because many new plugins require functionality in from the 0.5 dev tree. The problem is that just building `brew install neovim --HEAD` results in a broken binary, because 0.5 requires also luaJIT 2.1, which can only be installed with `brew install luajit --HEAD` AFAIK.

Should this be a requirement on the `HEAD` case in the formula? Or just a mention to install this dependency separately? Or should this be tribal knowledge one can find in closed issues (https://github.com/neovim/neovim/issues/13529)?

**Comment by SMillerDev** [comment]
> because many new plugins require functionality in from the 0.5 dev tree. 

This should be a reason for upstream to tag a release. If they don't want to tag a release and our release is too old to work with most plugins that would be a reason to remove neovim from homebrew/core for failing to match https://docs.brew.sh/Acceptable-Formulae#stable-versions

**Comment by carlocab** [comment, marked as answer]
I actually opened https://github.com/Homebrew/homebrew-core/pull/70053 a while ago to try to fix this.
